src/rules/function-url-data-uris/index.js

```diff
@@ -0,0 +1,53 @@
+import valueParser from "postcss-value-parser"
+import {
+  cssWordIsVariable,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "function-url-data-uris"
+
+export const messages = ruleMessages(ruleName, {
+  expected: "Expected data uris in function url",
+  rejected: "Unexpected data uris in function url",
+})
+
+export default function (expectation) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: expectation,
+      possible: [
+        "always",
+        "never",
+      ],
+    })
+    if (!validOptions) { return }
+
+    root.walkDecls(function (decl) {
+      valueParser(decl.value).walk(valueNode => {
```

Comment:
Do you think it would be feasible to come up with a list of properties that will accept `url()` values? If so, we could speed up this rule by ignoring all declarations that don't match those properties.